### ISSUE TITLE
More original sources for libtapi

### ISSUE THREAD

**Issue by Ericson2314** [tpoechtrager/apple-libtapi#6]
Have you see https://github.com/ributzka/tapi? From http://lists.llvm.org/pipermail/llvm-dev/2017-September/117264.html it sounds like this is a release by the original author. As you can see from fetching both repos and doing
```sh
git diff \
  'b9205695b4edee91000383695be8de5ba8e0db41' \
  '5dc2fda8884ca6a96dd7f81c40349d318ae5c1b5:src/apple-llvm/src/projects/libtapi'
```
It differs quite greatly from your original commit. Perhaps there's good stuff to incorporate here?

**Comment by Ericson2314** [comment]
BTW wrote http://lists.llvm.org/pipermail/llvm-dev/2018-April/122434.html responding to that thread in hopes there's still a chance it might be upstreamed. [Too bad I forgot to quote previous email...]

**Comment by Ericson2314** [comment]
https://reviews.llvm.org/D37820 actually there is this vageuly TAPI-related diff.

**Comment by tpoechtrager** [comment]
Thank you. Latest TAPI sources have been merged into the 2.0.0 branch.
